src/components/switch/examples/switch-readonly.tsx

```diff
@@ -42,15 +41,21 @@ export class SwitchReadonlyExample {
         return [
             <limel-switch
                 label="Subscribe to email newsletters"
-                readonlyProps={{
-                    trueIcon: 'news',
-                    falseIcon: {
-                        name: 'cancel_subscription',
-                        color: 'rgb(var(--color-orange-default))',
+                readonlyLabels={[
+                    {
+                        value: true,
+                        icon: 'news',
+                        text: 'Is subscribed to receive newsletters',
+                    },
+                    {
+                        value: false,
+                        icon: {
+                            name: 'cancel_subscription',
+                            color: 'rgb(var(--color-orange-default))',
+                        },
+                        text: 'Is unsubscribed from newsletters',
```

Comment:
https://github.com/Lundalogik/lime-elements/assets/35954987/9deb035f-ad49-4f6c-9ab2-24cbbc228563

these seem to work in a weird way for limel-switch. You have to toggle the selected state for them to work

Reply:
cc42437a8c21ff9e60fa86f940011d4ce5f90d81